thoth/package_extract/image.py

```diff
@@ -178,6 +179,32 @@ def _parse_deb_dependency_line(line_str: str) -> typing.List[tuple]:
     return result
 
 
+def gather_digests(path_: str) -> dict:
```

Comment:
It will be a good idea to rewrite this so that the resulting JSON produces a flattened list of files present (not nested dictionaries) with the full path to a file for which digest was computed. You don't need to use recursion for this case, you can use `os.walk` instead [1].

[1] https://docs.python.org/3/library/os.html#os.walk